worker/config.go

```diff
@@ -199,6 +201,18 @@ func ConfigSchema() schematypes.Object {
 				Minimum: 0,
 				Maximum: 65535,
 			},
+			"networkInterface": schematypes.String{
+				MetaData: schematypes.MetaData{
+					Description: "Network device webhookserver should listen on, if not serverIp",
+				},
+			},
+			"exposedPort": schematypes.Integer{
+				MetaData: schematypes.MetaData{
+					Description: "Port webhookserver should listen on if not serverPort",
```

Comment:
nit: "Port webhookserver should listen on. If not supplied, it uses the serverPort value."